mmseg/models/losses/dice_loss.py

```diff
@@ -68,6 +68,9 @@ class DiceLoss(nn.Module):
             str format, read them from a file. Defaults to None.
         loss_weight (float, optional): Weight of the loss. Default to 1.0.
         ignore_index (int | None): The label index to be ignored. Default: 255.
+        loss_name (str, optional): Name of the loss item. If you want this loss
+            item to be included into the backward graph, `loss_` must be the
+            prefix of the name. Defaults to 'loss_dice'.
```

Comment:
`loss_` must be the prefix of the name. Defaults to 'loss_dice'.

Why `loss_` must be the prefix of the name in MMGen? and what is the key for MMSeg?